proto/api/v1/api.proto

```diff
@@ -35,8 +36,11 @@ message Pipeline {
 
   // -- children connections ---------------------------------------------------
 
-  repeated string connector_ids = 5 [(google.api.field_behavior) = OUTPUT_ONLY];
-  repeated string processor_ids = 6 [(google.api.field_behavior) = OUTPUT_ONLY];
+  repeated string connector_ids = 4 [(google.api.field_behavior) = OUTPUT_ONLY];
+  repeated string processor_ids = 5 [(google.api.field_behavior) = OUTPUT_ONLY];
```

Comment:
As much as I'd also like to have numbers aligned, this would create a breaking change and anybody using the gRPC API would need to generate their client again. So for that reason, I think we shouldn't change the field numbers of existing fields, at least not now. Maybe we can clean them up before releasing v1.